doc/source/getting_started/installation.rst

```diff
@@ -1,10 +1,14 @@
-Installation
-############
+.. _ref_dev_mode:
 
-This page assumes that you want to install PyAnsys Geometry in developer mode so that
+Package installation in development mode
+########################################
+
+This n assumes that you want to install PyAnsys Geometry in developer mode so that
 you can modify the source and enhance it. You can install PyAnsys Geometry from PyPI
 or from the `PyAnsys Geometry repository <https://github.com/ansys/pyansys-geometry>`_ on GitHub.
```

Comment:
```suggestion
This topic assumes that you want to install PyAnsys Geometry in developer mode so that
you can modify the source and enhance it. You can install PyAnsys Geometry from PyPI
or from the `PyAnsys Geometry repository <https://github.com/ansys/pyansys-geometry>`_ on GitHub.
```